world-bank-app/src/components/Search.js

```diff
@@ -1,6 +1,7 @@
 import React, { useState, useEffect } from "react";
-import { Form, Button } from "react-bootstrap";
+import { Form, Button, Container, Navbar } from "react-bootstrap";
```

Comment:
Have you considered importing individually like 
`import Button from 'react-bootstrap/Button';`
Doing so pulls in only the specific components that you use, which can significantly reduce the amount of code you end up sending to the client.

Reply:
Sounds good, will do that now